
![CancerMap banner](/./main_files/CGIM-banner.png)

Almost two decades after sequencing the human genome, understanding the complex genetics of quantitative phenotypes remains a major challenge in biology and medicine. This is in large parts due to extensive redundancies, synergisms and context-dependent gene functions. Whilst recent efforts to map all genetic interactions (GIs) in yeast produced unprecedented biological insights, similarly comprehensive approaches in human cells lack feasibility so far, mainly due to (i) the large size of the human genome and (ii) limitations inherent to genome perturbation methods. Exploiting new library generation technologies, which we developed to increase CRISPR screening scalability, we propose here to perform comprehensive SpCas9-based mapping of genetic interactions in human cells. We will use ~34 million pairwise gRNA combinations to perturb 4.2 million human gene pairs in cellular models of colon, lung and pancreatic cancer, the three leading causes of cancer-related death in Western countries. To facilitate data interpretation and downstream functional work, we will focus our surveys on important cancer pathways (WNT, RAS, ubiquitin) and on clinical challenges (druggable genome), which are key areas of expertise in the three applicant laboratories. We will harness GI profiling for biological discovery and therapeutic target identification at multiple levels. First, genome-wide combinatorial surveys for GIs involving the WNT, RAS, and ubiquitin systems will give new insights into the composition and functional organization of these key oncogenic pathways. We will identify new pathway components, dissect synergisms, redundancies and hierarchies within pathways, uncover signaling modulators as well as cross-talk between pathways. Second, we will exploit GI mapping combined with analysis of interaction profile similarity to predict the participation of genes in signaling pathways or cellular processes. Coupled with spatial enrichment analyses of biological processes in networks, these efforts will systematically annotate the function or uncharacterized genes and predict novel function for known genes in an unbiased manner. Third, we will validate GIs across large cell line panels and entities to systematically explore molecular and cellular context-dependencies, ultimately aiming at better understanding tissue-specific differences in tumorigenesis and the organization of signaling pathways. Finally, our screens will probe more than one million interactions involving FDA-approved drug target genes across entities. We aim at uncovering new avenues to indirectly target undruggable mutations, as well as new synthetic vulnerabilities for combinatorial drug repurposing, which we will test in preclinical models. Our proposed project represents the largest genetic interaction screening effort in human cells to date. It promises to give multifaceted insights into the biological underpinnings of human cancer and to uncover new avenues for therapeutic intervention.

*Etiam auctor dolor massa, in lobortis ex placerat et. Mauris at odio vulputate, eleifend massa at, pharetra magna. Phasellus cursus nisi libero. Integer posuere magna id iaculis mattis. Nam sed ultrices magna. Cras luctus placerat nunc, placerat volutpat leo. Donec luctus blandit ex, ut vehicula massa venenatis ac. Phasellus pulvinar, nibh eu ultrices ullamcorper, magna ex maximus metus, a mollis sem magna et eros. Integer tristique varius sem nec ultrices. Mauris nisi risus, vulputate vitae ornare non, vulputate sit amet nulla. Maecenas velit lorem, luctus vulputate blandit sagittis, finibus non urna. Pellentesque tempor euismod maximus. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas.*
